### PR TITLE
Limit max bounty to 99%

### LINKED ISSUE #208: shares logic will fail if paid out bounty of 100%

### PR DESCRIPTION
**Pull request by ben-kaufman**
Limits the max bounty to 99% of the vault funds ( Fix #208 )

### CODE REVIEW

**Review by jellegerbrandy — COMMENTED** [review]
Did we not say 90%?

File: contracts/vaults/Base.sol [hunk 1]
```diff
@@ -121,6 +121,7 @@ contract Base is ERC4626Upgradeable, ReentrancyGuardUpgradeable {
     }
 
     uint256 public constant HUNDRED_PERCENT = 10000;
+    uint256 public constant MAX_BOUNTY_LIMIT = 9900; // Max bounty can be uup to 99%
```
Comment (lirona) [inline contracts/vaults/Base.sol:124]:
```suggestion
    uint256 public constant MAX_BOUNTY_LIMIT = 9900; // Max bounty can be up to 99%
```

**Review by lirona — COMMENTED** [review]
lgtm

**Comment by lirona** [comment]
> Did we not say 90%?

shay said 99%

**Review by jellegerbrandy — APPROVED** [review]

**Pull request merged**